CHANGELOG.md

```diff
@@ -1,3 +1,25 @@
+## [9.2.0](https://github.com/dynatrace-oss/barista/compare/9.1.0...9.2.0) (2021-02-23)
+
+
+### Bug Fixes
+
+* **combobox:** Fixes an issue with the arrow positioning on safari. ([a3f2535](https://github.com/dynatrace-oss/barista/commit/a3f25352e67c48b124a6999ec8dd299820216bf4))
+* **expandable-section:** Fixes an issue where toggling the expandable section was triggering a form submission. ([8388bd0](https://github.com/dynatrace-oss/barista/commit/8388bd02875e436bbe8ab26c006edada49a4a5cb))
+* **filter-field:** Improving isOptionSelected method for async values. ([8e352fa](https://github.com/dynatrace-oss/barista/commit/8e352fa61173307a56b42c1646ee33c18e17cc73))
+* **filter-field:** Splits disabled into two members by adding a temporarilyDisabled member. This bool value relates to the edit mode, disabling intractions with existing filters while in edit mode. This is visualized with a not-allowed cursor. ([54dff6e](https://github.com/dynatrace-oss/barista/commit/54dff6e11eff75001c30bc44f8c42f5de77d1d3d))
```

Comment:
I think we can remove this since this is only internal information and not exposed to the consumer.

Reply:
```suggestion
```

Reply:
This changes the filter jumping behavior. Maybe we should reword it?